scrnaseq_hto.Rmd

```diff
@@ -7,26 +7,40 @@ geometry: margin=2cm
 output:
    html_document:
     toc: true
+    toc_depth: 4
```

Comment:
For later: If we stick to the floating toc, we can introduce it here as well.

Reply:
Already done